src/Extension/FlashMessageRuntime.php

```diff
@@ -36,17 +36,22 @@ public function __construct(FlashManager $flashManager)
     /**
      * Returns flash messages handled by Sonata flash manager.
      *
-     * @param string $type Type of flash message
+     * @param string      $type             Type of flash message
+     * @param string|null $deprecatedDomain Translation domain to use
+     *
+     * @return string
```

Comment:
IIUC, this method has a pluralized name and returns a single string?

Reply:
Wrong core-bundle phpdoc.